src/patternfly/components/Popover/popover.scss

```diff
@@ -76,6 +76,11 @@
   padding: var(--pf-c-popover__content--PaddingTop) var(--pf-c-popover__content--PaddingRight) var(--pf-c-popover__content--PaddingBottom) var(--pf-c-popover__content--PaddingLeft);
   background-color: var(--pf-c-popover__content--BackgroundColor);
 
+  > .pf-c-title {
+    margin-bottom: var(--pf-c-popover--c-title--MarginBottom);
+    line-height: 1;
```

Comment:
When I inspect this, it looks like the line-height from the title component (1.5) is overriding the line-height (1) that is set here. Do you see that too?

Reply:
Ah nice catch, I don't know why `line-height: 1` is there. If the title wraps to 2 lines, there isn't enough space/leading. I'm going to remove that :)

<img width="347" alt="Screen Shot 2019-05-24 at 3 35 33 PM" src="https://user-images.githubusercontent.com/35148959/58355390-d13a3800-7e39-11e9-8378-8620750ef5ae.png">